README.md

```diff
@@ -62,6 +62,22 @@ The result looks like this:
 
 If Mercury is unable to find a field, that field will return `null`.
 
+## `parse()` Options
+
+By default, Mercury Parser will parse all pages of a page and return the cleansed content as HTML. However, you can override this behavior by passing in options to the `parse` function, specifying whether or not to scrape all pages of an article, and what type of output to return (valid values are `'html'`, `'markdown'` and `'text'`). For example:
+
+```javascript
+Mercury.parse(url, { contentType: 'markdown' }).then(result => console.log(result));
+```
+
+This returns the the page's `content` as GitHub-flavored Markdown:
+
+```json
+"content": "...**Thunder** is the [stage name](https://en.wikipedia.org/wiki/Stage_name) for the..."
+```
+
+## The command-line parser
+
 Mercury Parser also ships with a CLI, meaning you can use the Mercury Parser
 from your command line like so:
```

Comment:
Since we're documenting above, can you also just add those options in the bash section below. E.g.:

```bash
# Install Mercury globally
yarn global add @postlight/mercury-parser
#   or
npm -g install @postlight/mercury-parser

# Then
mercury-parser https://postlight.com/trackchanges/mercury-goes-open-source

# Pass optional --format argument to set content type
mercury-parser https://postlight.com/trackchanges/mercury-goes-open-source --format=markdown
```